### PR TITLE
Dont inline external functions

### PR DESCRIPTION
**Pull request by dneto0**

### CODE REVIEW

**Comment by dneto0** [comment]
Cc: @greg-lunarg

File: source/opt/inline_pass.h [hunk 1]
```diff
@@ -117,6 +117,9 @@ class InlinePass : public Pass {
                      ir::UptrVectorIterator<ir::Instruction> call_inst_itr,
                      ir::UptrVectorIterator<ir::BasicBlock> call_block_itr);
 
+  // Returns true if |inst| is a function call that can be inlined.
+  bool IsInlinableFunctionCall(const ir::Instruction* inst);
```
Comment (antiagainst) [inline source/opt/inline_pass.h]: const function?

Comment (dneto0) [inline source/opt/inline_pass.h]: The unordered map access causes it to be non-const.  I didn't bother to rewrite it with the iterator getter.  There's another similar id lookup in the GenInlineCode function that is the same.

Comment (antiagainst) [inline source/opt/inline_pass.h]: Right... The const version of `operator[]` is `at`... But never mind.

**Pull request merged**